src/App.tsx

```diff
@@ -76,18 +66,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {selectedUser && loaded && !hasError && posts.length === 0 && (
```

Comment:
```suggestion
                {selectedUser && loaded && !hasError && !posts.length && (
```
see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)